phpfarm/docker.sh

```diff
@@ -17,14 +17,23 @@ do
 
     # compile the PHP version
     ./compile.sh $VERSION
+
+    # PHP 5.1 names both CLI and CGI executables 'php'. Rename and move them to
+    # correct dirs. See http://php.net/manual/bg/features.commandline.php
+    if [ "$V" == "5.1" ]; then
+        post='custom/post-install-5.1.6.sh'
+        echo "Running commands from '$post'"
+        /bin/bash "$post" "$VERSION"
+    fi
```

Comment:
So after the build completes, in _/phpfarm/inst/php-5.1.6/bin/_ you'll only see:
* php
* php-config
* phpize

However the post-install script fixes this by re-copying the correct binaries from the PHP source dir, see the [executable copying section in _post-install-5.1.6.sh_](https://github.com/splitbrain/docker-phpfarm/pull/13/files/e1c805c6aada4924094e744447a2aba5fd66e383#diff-d60c9fa50dd30cbe7e7b80e311173ab7R15).